cxx-qt-gen/test_outputs/handlers.h

```diff
@@ -1,10 +1,13 @@
 #pragma once
 
+#include <memory>
 #include <mutex>
 
-#include "cxx-qt-lib/include/qt_types.h"
+namespace cxx_qt::my_object {
+class MyObject;
+} // namespace cxx_qt::my_object
 
-#include <QtCore/QString>
+#include "cxx-qt-gen/include/my_object.cxx.h"
```

Comment:
Do we still need to forward declare the Rust side of the MyObject? as that's included from the .cxx.h?

Reply:
We don't, removed :-)